WalletWasabi.Gui/Converters/LurkingWifeModeStringConverter.cs

```diff
@@ -28,7 +28,7 @@ public object Convert(object value, Type targetType, object parameter, CultureIn
 
 		public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
 		{
-			throw new NotSupportedException();
+			return value.ToString();
```

Comment:
Shouldn't it be better to do this? (It's a question.)

```suggestion
			return value?.ToString() ?? "";
```

Reply:
Yes I agree.

Reply:
Iv tested this change also FWIW